language-design.md

```diff
@@ -59,7 +252,35 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
-
+First of all, the post on grayscale shows the value of collaborating with users
+to design an API.  One of Bloch's maxims was that "API design is not a
+solitary activity," and the authors demonstrated this importance.  The
+article first listed what seemed like an extensive list of pros and cons for the
+function names.  Other people then commented with additional suggestions that
+the authors perhaps hadn't considered.  For example, one of the comments
+suggested avoiding `black(lightness [, alpha])` because it involved a
+"subtractive model of printing".  This commenter provided valuable insight on
+an issue that was not previously mentioned in the article.
+
+Secondly, the article shows the different factors that coders consider when
+choosing a function's name.  First of all, coders should obey the Principle of
+Least Astonishment: "Every method should do the least surprising thing it
+could, given its name" [Bloch, 2006].  The writers of the API went to great
+lengths to provide the least astonishing name for their function by polling
+users to gauge their intuition.  They also considered whether small changes in
+behavior would require extensive changes in the code. This provided
+encouragement for `rgb(x)`, since if you later wanted to change the color away
+from gray, you would only have to change the arguments rather than the function
+name.  Furthermore, readability was a big concern for naming the grayscaling
+function.  Although `rgb(x)` may have its benefits, it is much less readable
+than, for example, `gray(x)`.
```

Comment:
I agree with this. Verou argues that since programmers are accustomed to using `rgb()`, it would be beneficial to use the same function name for grayscale. However, if I saw a function call to `rgb()` that only uses one argument, I wouldn't immediately understand that it's for grayscale.